docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -649,6 +649,14 @@ Connect ID Token provider such as Google.
 }
 ```
 
+#### Importing social sign-in connections with email
+
+If your system doesn't have a `subject` ID, you can use the email address as the subject. This isn't recommended, as the email
```

Comment:
This is not quite true. 

The email address is used to match the incoming SSO request to an identity and if successful, we use the subject on subsequent logins. 

As it stands, it seems like we will always use the email as the subject for these identities.

Reply:
Do we not import user with an OIDC provider email like this?

```
{
  "provider": "google",
  "use_auto_link": true,
  "subject": "the@email.org"
}
```

And on account linking it is converted to

```
{
  "provider": "google",
  "use_auto_link": false,
  "subject": "1234"
}
```

Reply:
I see, the section is misleading.

Reply:
fixed